pydis_site/templates/events/sidebar/events-list.html

```diff
@@ -1,10 +1,10 @@
 <div class="box">
```

Comment:
What is this by the way?
![image](https://user-images.githubusercontent.com/38372706/134391486-8e3fd116-3e15-4769-9af3-d8a782d89775.png)

It is on the production site as well: https://pythondiscord.com/events/

Reply:
It's the [breadcrumb](https://bulma.io/documentation/components/breadcrumb/) for the page.